_posts/2017-07-29-css-grids.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+author: Cyan/Jos
+title: Grid CSS
+---
+
+### What is CSS Grids?
```

Comment:
Stick with managing header tags in a hierarchical fashion. i.e., post title is H1 (`#`), subtitle H2 (`##`)

If you don't like their resulting size, then that's something to tweak in CSS, not by using markdown